MapleServer2/MapleServer.cs

```diff
@@ -11,6 +11,8 @@
 
 namespace MapleServer2 {
     public static class MapleServer {
+
+        private static GameServer gameServer;
```

Comment:
If you want to use him, instead declare him on gamesession as `private readonly static GameServer gameServer;`